db/db_impl/db_impl.cc

```diff
@@ -485,10 +489,15 @@ Status DBImpl::CloseHelper() {
     }
   }
 
-  if (default_cf_handle_ != nullptr) {
+  if (default_cf_handle_ != nullptr || persist_stats_cf_handle_ != nullptr) {
     // we need to delete handle outside of lock because it does its own locking
     mutex_.Unlock();
-    delete default_cf_handle_;
+    if (default_cf_handle_) {
+      delete default_cf_handle_;
```

Comment:
Set both `default_cf_handle_` and `persist_stats_cf_handle_` to `nullptr` after deletion?